src/_rules/typography.js

```diff
@@ -10,4 +10,6 @@ export const typography = [
     ({ 'line-height': `var(--w-line-height-${size})` }),
   ],
   [/^leading-\[(.+)(rem|px)?\]/, ([, value, unit], context) => ({ 'line-height': resolveArbitraryValues(value, unit, context) })],
+  ['tabular-nums', { 'font-family': 'ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif', 'font-variant-numeric': 'tabular-nums' }],
+  ['lining-nums', { 'font-variant-numeric': 'lining-nums' }],
```

Comment:
Used in Finn, and not in Warp yet. It's from [tailwind](https://tailwindcss.com/docs/font-variant-numeric).